### PR TITLE
chore(breadcrumb-item): fix anchor font style

### PR DESCRIPTION
**Pull request by yinonov**

### CODE REVIEW

File: libs/components/src/lib/breadcrumb/breadcrumb.spec.ts [hunk 1]
```diff
@@ -38,6 +38,9 @@ describe('vwc-breadcrumb', () => {
 	});
 
 	describe('aria-current', function () {
+		/**
+		 *
+		 */
```
Comment (yinonov) [inline libs/components/src/lib/breadcrumb/breadcrumb.spec.ts:43]:
```suggestion
```

File: libs/components/src/lib/breadcrumb/ui.test.ts [hunk 2]
```diff
@@ -8,6 +8,9 @@ import {
 
 const componentName = 'breadcrumb';
 
+/**
+ * @param page
+ */
```
Comment (yinonov) [inline libs/components/src/lib/breadcrumb/ui.test.ts:13]:
```suggestion
```

**Review by YonatanKra — DISMISSED** [review]
LGTM. Why did `layout`'s snapshot change?

**Comment by yinonov** [comment]
> LGTM. Why did `layout`'s snapshot change?

it always changes. not sure

**Review by YonatanKra — APPROVED** [review]

**Pull request merged**